packages/airnode-deployer/src/infrastructure/index.ts

```diff
@@ -688,3 +689,75 @@ export async function deploymentInfo(deploymentId: string) {
   const tableStringWithCurrent = tableString.replace(new RegExp(`(?<=${currentVersionId}.*?)\n`), ' (current)\n');
   consoleLog(tableStringWithCurrent);
 }
+
+export async function fetchFiles(deploymentId: string, outputDir: string, versionId?: string) {
```

Comment:
Nit: Maybe use `try finally` to stop the spinner at the end of the function execution instead of doing it manually?

Reply:
Can't really do that because I need to call `spinner.stop` in case of a failure or `spinner.succeed` if it's successful. I would need to have a `catch` block, catching the exceptions I'm throwing, stopping the spinner, and immediately rethrowing them. And the whole function would need to be in a `try catch finally` block, that seems weird. I'd rather keep it this way.

Reply:
👍 Makes sense. I didn't notice you need to call a different one if the operation was successful.